packages/itensor/tests/itensor/estimator/test_itensor_estimator.py

```diff
@@ -0,0 +1,180 @@
+import math
+from concurrent.futures import ProcessPoolExecutor
+from multiprocessing import get_context
+from typing import Union
+
+import pytest
+
+from quri_parts.circuit import (
+    UnboundParametricQuantumCircuit,
+    UnboundParametricQuantumCircuitProtocol,
+)
+from quri_parts.core.operator import Operator, PauliLabel, pauli_label
+from quri_parts.core.state import ComputationalBasisState, ParametricCircuitQuantumState
+from quri_parts.itensor.estimator import (
+    create_itensor_mps_concurrent_estimator,
+    create_itensor_mps_estimator,
+    create_itensor_mps_parametric_estimator,
+)
+
+
+class TestITensorEstimator:
+    def test_estimate_pauli_label(self) -> None:
+        pauli = pauli_label("Z0 Z2 Z5")
+        state = ComputationalBasisState(6, bits=0b110010)
+        estimator = create_itensor_mps_estimator()
+        estimate = estimator(pauli, state)
+        assert estimate.value == -1
+
+    def test_estimate_operator(self) -> None:
+        operator = Operator(
+            {
+                pauli_label("Z0 Z2 Z5"): 0.25,
+                pauli_label("Z1 Z2 Z4"): 0.5j,
+            }
+        )
+        state = ComputationalBasisState(6, bits=0b110010)
+        estimator = create_itensor_mps_estimator()
+        estimate = estimator(operator, state)
+        assert estimate.value == -0.25 + 0.5j
+
+
+class TestITensorConcurrentEstimator:
```

Comment:
I took profiling of the slow test cases but only found that `{method 'acquire' of '_thread.lock' objects}` spends all the time...
Until we figure out the reason it may be better to skip those test cases involving concurrent execution with `ProcessPoolExecutor`.

![image](https://user-images.githubusercontent.com/3595560/227516831-722182c3-65f3-4286-ab5a-c7f4ae0c2a01.png)